docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if chown -R www-data:www-data /var/www/html/ ; then
+  echo "All files were chown'ed properly"
+else
+  echo "There were some errors while chown'ing files"
+fi
+docker-php-entrypoint apache2-foreground
```

Comment:
Small modification to the entrypoint. Now it only checks the folder for permission which are required for the server to write in. When I am using the container for development, it is nice to still be able to write and replace php files.

```suggestion
#!/usr/bin/env bash

if test -f /var/www/html/inc/conf.php; then
  if chown www-data:www-data /var/www/html/inc/conf.php ; then
    echo "conf.php was chown'ed properly"
  else
    echo "There were some errors while chown'ing conf.php"
  fi
fi

if chown -R www-data:www-data /var/www/html/import/ ; then
  echo "import folder was chown'ed properly"
else
  echo "There were some errors while chown'ing the import folder"
fi

if chown -R www-data:www-data /var/www/html/files/ ; then
  echo "files folder was chown'ed properly"
else
  echo "There were some errors while chown'ing files folder"
fi

docker-php-entrypoint apache2-foreground
```